### ISSUE TITLE
NVDA does not support attribute aria-errormessage

### ISSUE THREAD

**Issue by kranid** [nvaccess/nvda#7768]
### Steps to reproduce:

Go to any text field with attribute aria-errormessage

### Expected behavior:
NVDA should announces the error message. 
### Actual behavior:
NVDA does not announce the error message
### System configuration:
NVDA version:
NVDA latest  stable version +firefox

**Comment by PratikP1** [comment]
Can you provide sample code or a live form where aria-errormessage is used? As aria 1.1 is a proposed recommendation, the use of aria-errormessage may be limited.

Technical: For future implementation, Here is a reference  to the aria-errormessage property spec: https://www.w3.org/TR/wai-aria-1.1/#aria-errormessage

**Comment by ehollig** [comment]
Closing as duplicate of #8318, as it contains more information.
